litmus-portal/frontend/src/components/LitmusStepper/styles.ts

```diff
@@ -51,16 +51,16 @@ const useStyles = makeStyles((theme) => ({
 
 const useStepIconStyles = makeStyles((theme) => ({
   root: {
-    display: 'flex',
     alignItems: 'center',
-    justifyContent: 'center',
-    width: '1.5rem',
-    height: '1.5rem',
+    background: theme.palette.text.hint,
     borderRadius: '50%',
+    color: theme.palette.text.primary,
```

Comment:
This doesn't seem readable to me...previously all the display properties and flex related properties were together and width and height were together...sorting it alphabetically makes it less readable for me. For example, after sorting alignItems comes before display flex, this won't cause any CSS error and CSS will work just fine...but to a developer, it makes less sense I believe.

Reply:
Sure @arkajyotiMukherjee, I agree too.
No issues, this was just I case we want to enforce Alphabetically order for CSS properties.